New Weblate documentation (#381)
AlmaLinux/wiki: docs/Help-translating-site.md

---
title: 'Help translating site'
---
# Help translating the site

> The [almaLinux.org websites](https://almalinux.org/) localization and translation is managed by [Weblate](https://hosted.weblate.org/engage/almalinux/) and supported by contributors, the translations that get submitted through Weblate are automatically submitted to the [almaLinux.org repo](https://github.com/AlmaLinux/almalinux.org) as a pull request.

Reviewer: ```suggestion
The [almalinux.org website](https://almalinux.org/) localization and translation is managed by [Weblate](https://hosted.weblate.org/engage/almalinux/) and supported by contributors. The translations that get submitted through Weblate are automatically submitted to the [almalinux.org repo](https://github.com/AlmaLinux/almalinux.org) as a pull request.
```

> > Pull requests are reviewed by a member of the marketing SIG or another team lead, and merged as appropriate.

Reviewer: ```suggestion
Pull requests from Weblate are reviewed by a member of the marketing SIG or another team lead, and merged as appropriate.

If you would like to help contribute to translations, you can do so by following the directions below!
```

## Create an account

> First you need to create a [Weblate](https://hosted.weblate.org/accounts/register/) account, you could use accounts from other service providers or fill in the form shown below to create it.

Reviewer: ```suggestion
First you need to create a [Weblate](https://hosted.weblate.org/accounts/register/) account. You could use accounts from other service providers or fill in the form shown below to create it.
```

![image](/images/weblate-create-account.png)

## Starting to translate

After creating an account we need to head to [AlmaLinux Foundation Weblate](https://hosted.weblate.org/projects/almalinux/):

![image](/images/weblate-almalinux-foundation.png)

> And choose each one of the two and you should be sent to a page similar to the image below.

Reviewer: ```suggestion
Choosing each one of the two should send you to a page similar to the image below.
```

![image](/images/weblate-translate-website.png)

> Then choose the language you want to translate to, for this example I'll choose [Hebrew](https://hosted.weblate.org/translate/almalinux/website-backend/he/?q=state:%3Ctranslated).

Reviewer: ```suggestion
Then choose the language you want to translate to. For this example, I'll choose [Hebrew](https://hosted.weblate.org/translate/almalinux/website-backend/he/?q=state:%3Ctranslated).
```

![image](/images/weblate-send-translations.png)

> After translating from English to your chosen language (.e.g Hebrew) press on "Save and continue" and your translation would be sent eventually as a PR to the [almaLinux.org repo](https://github.com/AlmaLinux/almalinux.org) and reviewed by a member of the marketing SIG or another team lead, then merged as appropriate.

Reviewer: ```suggestion
After translating from English to your chosen language (.e.g Hebrew) press on "Save and continue" and your translation will be sent eventually as a PR to the [almalinux.org repo](https://github.com/AlmaLinux/almalinux.org) and reviewed by a member of the marketing SIG or another team lead, then merged as appropriate.
```
